aws-iot-scheduledaudit/inputs/inputs_1_invalid.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
nit: Maybe rename this file to suggest that this test is for CREATE path only?

Reply:
The format for these names is mandated by the CFN framework, unfortunately.